backend/handler/passcode.go

```diff
@@ -377,6 +382,19 @@ func (h *PasscodeHandler) Finish(c echo.Context) error {
 			return fmt.Errorf("failed to create audit log: %w", err)
 		}
 
+		// notify about email verification result. Last step to prevent a trigger and rollback scenario
+		if h.cfg.Emails.RequireVerification && wasUnverified {
+			var evt events.Event
+
+			if len(user.Emails) >= 1 {
```

Comment:
Unfortunately this check does not work, because `user.Emails` is set before, when the user has no emails https://github.com/teamhanko/hanko/blob/38209aa1bdf4c40edf634455594bb899e821e86c/backend/handler/passcode.go#L348 and because of that the check is always `true`.

Reply:
should be fixed now. I moved the check to the same level as `wasUnverified` and am only using the result for the trigger if else